packages/eslint-config-react-app/package.json

```diff
@@ -21,9 +21,12 @@
     "eslint": "6.x",
     "eslint-plugin-flowtype": "3.x",
     "eslint-plugin-import": "2.x",
+    "eslint-plugin-jest": "23.x",
+    "eslint-plugin-jest-dom": "1.x",
     "eslint-plugin-jsx-a11y": "6.x",
     "eslint-plugin-react": "7.x",
-    "eslint-plugin-react-hooks": "1.x"
+    "eslint-plugin-react-hooks": "1.x",
+    "eslint-plugin-testing-library": "1.x"
```

Comment:
v2 was released recently, removing a controversial rule enabled by default. I think should be upgraded to v2.x

Reply:
Thanks for this, just updated.